README.md

```diff
@@ -39,7 +39,7 @@ This application uses [HMPPS Auth](https://github.com/ministryofjustice/hmpps-au
 
 To seed data run `npm run seed-dev-data`, this will add users with elevated roles. To login to a user which has a specific role please use the following usernames:
 
-- WMT_DATA_ADMIN
+- WMT_SUPER_USER
```

Comment:
I think we will need to change hmpps-auth for this change to work when running locally against docker. You can currently log in as WMT_SUPER_USER (to my surprise) but you do not have admin privileges

I have since found you can log in locally as WMT_LITERALLY_ANYTHING as long as you use the correct password! Not clear on how or why this works yet.

Reply:
we use wiremock to mock Delius https://github.com/ministryofjustice/wmt-web/blob/main/test/e2e/resources/wiremock/mappings/user-detail-success.json so currently anything beginning with WMT as a username will work. Then to give elevated roles we use https://github.com/ministryofjustice/wmt-web/blob/main/test/helpers/routes/authentication-helper.js#L2 to populate the database through the `seed-dev-data` mechanism